packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -89,25 +92,30 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     ///         Set as internal with a getter so that the struct is returned instead of a tuple.
     ResourceMetering.ResourceConfig internal _resourceConfig;
 
+    /// @notice The block at which the op-node can start searching for logs from.
+    uint256 public startBlock;
+
+    /// @notice SuperchainConfig address
+    ///         This can only be set via upgrade.
+    SuperchainConfig public superchainConfig;
+
+    /// @notice Semantic version.
+    /// @custom:semver 2.0.0
+    string public constant version = "2.0.0";
+
```

Comment:
This diff moves the ConfigUpdate event below the variable declarations.
It should not cause issues with the storage layout.